src/services/services.ts

```diff
@@ -259,11 +259,11 @@ namespace ts {
         }
 
         public getChildCount(): number {
-            return 0;
+            return this.getChildren().length;
         }
 
-        public getChildAt(): Node {
-            return undefined!;  // TODO: GH#18217
+        public getChildAt(index: number): Node {

```

Comment:
Should return `Node | undefined`?
TypeScript's codebase chould use the stricter tsconfig flags...

Reply:
Theoretically, yes, but that's somewhat tangential to this PR